src/ashwini/assignment16/FrequencyOfCharecterInString.java

```diff
@@ -0,0 +1,35 @@
+package ashwini.assignment16;
+
+import java.util.Scanner;
+
+public class FrequencyOfCharecterInString {
+	int count ;
+	char character;
```

Comment:
You should write this instance variable with in method as, it’s useful for only that method.  If variable going to use multiple places then it makes senses to define that variable at instance. just trying to explain good coding practices.